components/mailbox/src/Mailbox.svelte

```diff
@@ -304,20 +304,24 @@
     }
   }
 
```

Comment:
Renaming this to `handleEmailDraftOpened` since it's handling all the events coming from `Email` component that triggers a draft opening.

What this function does is checking if the event is opening a draft. And if yes, fetch its body and pass on the event for dispatch.